app/config/config_dev.yml

```diff
@@ -14,15 +14,18 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  %kernel.logs_dir%/%kernel.environment%.log
-            level: debug
+            type:   stream
+            path:   %kernel.logs_dir%/%kernel.environment%.log
+            level:  debug
+        console:
+            type:   console
+            bubble: false
```

Comment:
it doesnt really matter if bubble is enabled or not as the firephp and chromephp handler are not enabled anymore by default in SE.